roadmap/implementers-guide/src/node/disputes/participation.md

```diff
@@ -0,0 +1,94 @@
+# Dispute Participation
+
+Tracks all open disputes in the active leaves set.
+
+## Remarks
+
+`UnsignedTransaction`s are OK to be used, since the inner
+element `CommittedCanddidateReceipt` is verifiable.
+
+## ToDo
+
+1. Specify what is being gossiped, what messages and what they must contain.
+
+
+## IO
+
+Inputs:1. Receive `DisputeParticipationMessage::Resolution`
+1. Delete all relevant data associated with this dispute
+
+
+* `DisputeParticipationMessage::Detection`
+* `DisputeParticipationMessage::Resolution`
```

Comment:
Not yet specified: brief steps to take on `DisputeParticipationMessage::Detection`

Redundantly specified: `DisputeParticipationMessage::Resolution`